packages/evolution-generator/src/scripts/generate_widgets.py

```
@@ -782,6 +783,20 @@ def generate_choices(choices):
     return f"{INDENT}choices: choices.{choices}"
 
 
+def generate_additional_choices(choices, comma=True, skip_line=True):
```

Ruff (0.16.2)

[warning] Boolean default positional argument in function definition

(FBT002)